README.md

```diff
@@ -62,10 +62,8 @@ Options:
   -S, --silent                     If using npm, don't save in package.json
   -Y, --yarn                       Install with Yarn
   -P, --pnpm                       Install with pnpm
-  -r, --registry <uri>             Install from custom registry (defaults to NPM registry)
+  -n, --no-registry                Do not use a remote registry to find dependencies list
   --dry-run                        Do not install packages, but show the install command that will be run
-  -a, --auth <token>               Provide an NPM authToken for private packages.
-  -p, --proxy <http_proxy>         Enable http proxy to connect to the registry
```

Comment:
is there a way we can keep this functionality? the default can (should) still be the behavior in this PR, and `-n` is a good addition regardless.

Reply:
I'm not following, that's the point of this PR - remove the custom proxy logic that is unnecessary and doesn't work with many proxy configurations, offloading it to the package manager instead (see my comment on #72). If you have to deal with proxies, you are already having to deal with them in the package manager. Making users deal with them twice is unnecessary pain. I can't have the new behavior as default, while preserve the old behavior in a non-breaking way, because the old behavior was inconsistent with multiple behaviors hidden behind a single flag, (see #84).

Agreed that it's a breaking change and that's why it is flagged as such. Semantic-release should appropriately bump a major version number by design.

Reply:
Gotcha, that makes sense.